src/sqlparser/src/ast/statement.rs

```diff
@@ -850,6 +851,42 @@ impl fmt::Display for CreateConnectionStatement {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct CreateSecretStatement {
+    pub if_not_exists: bool,
+    pub secret_name: ObjectName,
+    pub credential: Value,
+    pub with_properties: WithProperties,
+}
+
+impl ParseTo for CreateSecretStatement {
```

Comment:
We may need to add some parser tests as well. Refer: https://github.com/risingwavelabs/risingwave/blob/main/src/sqlparser/README.md